Sync Canary -> Otbr

beats-dh opened the pull request:
https://github.com/opentibiabr/canary/commit/3e94dc2b6283a0c179bf2c628ea345aa5495c15e

newacc12 commented:
@dudantas While I do understand that the goal of this project is to keep as close as possible to global tibia, I think we should consider keeping the trainers. Nowadays, this is widely accepted and very popular functionality.

Thoughts?

dudantas commented:
I think it is very simple for those who have custom servers to create. I don't think it's necessary to be in the repository, because it's very easy to do that and I noticed that most people have their own custom trainers.

There are currently offline trainers and training dummy, I really don't see the need for custom trainers.

beats-dh committed:
[Fix] multiple purchase of the same mount from store.

https://github.com/opentibiabr/canary/commit/3e94dc2b6283a0c179bf2c628ea345aa5495c15e

beats-dh committed:
Update GHA

beats-dh committed:
Add tag serverSaveTime on config.lua (#251)

beats-dh committed:
Update discord_specialwebhook.lua
